src/EnergyPlus/SetPointManager.cc

```diff
@@ -3061,10 +3072,6 @@ namespace SetPointManager {
 		rNumericArgs.deallocate();
 		lNumericFieldBlanks.deallocate();
 
-		if ( ErrorsFound ) {
-			ShowFatalError( RoutineName + "Errors found in input.  Program terminates." );
-		}
-
```

Comment:
Moved this fatal out to wrapper GetSetPointManagerInputs to allow unit tests to call GetSetPointManagerInputData (new name for the GetInput).